new_platforms/vscode-extension/README.md

```diff
@@ -10,61 +10,87 @@ Ensure that the [requirements](#Requirements) are met.
+Visual Studio Code [Remote-Development extension](https://code.visualstudio.com/docs/remote/remote-overview) to utilize 
+[Standalone](#Standalone%20projects) projects on a Windows desktop.  In this case, you will need to install the [requirements](#Requirements) in
+the subsystem.
+
+## Standalone projects
+
 ### Build your Open Enclave solution.
 
-There are build tasks for both standalone and Azure IoT Edge container projects.  The underlying system used 
-to build is CMake.  
+The underlying system used to build is CMake.  
 
-For a standalone project, there will be configure and build tasks for each target (ARMv7-A and AArch64/ARMv8-A).  The 
+There will be configure and build tasks for each target (ARMv7-A, AArch64/ARMv8-A, and LS1012).  The 
 configure task will invoke CMake to create the required build files.  This is only required to be run once.  
 The build task will do the actual compiling and linking.
 
 1. **F1** or **CTRL-Shift-P**
 1. Select `Tasks: Run Task`
-1. Select `Configure for QEMU (ARMv7-A | AArch64/ARMv8-A)`
-1. Select `Build for QEMU (ARMv7-A | AArch64/ARMv8-A)`
+1. Select `Configure for QEMU (ARMv7-A | AArch64/ARMv8-A | LS1012)`
+1. Select `Build for QEMU (ARMv7-A | AArch64/ARMv8-A | LS1012)`
+
+### Debug your Open Enclave solution.
 
-For an Azure IoT Edge container project, containers are used to configure and build.  The build task in a
-container project will invoke docker and leverage project dockerfiles.  The container can be built by:
+Debugging your standalone project's enclave is easy.  
+
+1. Set breakpoints in the files you wish to debug.  Breakpoints in the enclave may only be added before
+the emulator (QEMU) starts or when the debugger is already broken inside the enclave.
+1. Choose the architecture you are interested in debugging by navigating to the Visual 
+Studio `Debug` view (**CTRL-Shift-D**) and selecting either `(gdb) Launch QEMU (ARMv7-A)`, 
+`(gdb) Launch QEMU (AArch64/ARMv8-A)`, or `(gdb) Launch QEMU (LS1012)` from the debug configuration dropdown.
+1. You can simply hit `F5`.  This will run cmake configuration, run the build, start QEMU, and load 
+the host and enclave symbols into an instance of the debugger.
+1. Open the **Terminal** view
+1. Log into QEMU using `root` (no password is required)
+1. Start the host process by entering `/mnt/host/bin/<solution-name>`
+
+        Note: The debugger has been configured to break at TA_InvokeCommandEntryPoint.  This will happen once when the enclave starts and once for each ECALL.
+
+## Azure IoT Edge projects
+
+### Build your Open Enclave solution.
+
+Ubuntu containers are used to configure and build.  The build task will invoke docker and leverage project 
+dockerfiles.  The container can be built by:
 
 1. Right click on `modules/<solution-name>/module.json`
 1. Select `Build IoT Edge Module Image`
-1. Select desired architecture and configuration from picker
+1. Select `ls1012`, `arm32v7-qemu` or `aarch64-qemu` from Platform picker
```

Comment:
Grammar: the lead sentence reads "[...] can be built by:", so the list should have its verbs in present participle form.

Reply:
ok